framework/src/mesh/MooseMesh.C

```diff
@@ -2328,6 +2334,12 @@ MooseMesh::needsPrepareForUse()
   _needs_prepare_for_use = true;
 }
 
+void
+MooseMesh::needsPrepareForUse(bool & needs_prepare_for_use) const
+{
+  needs_prepare_for_use = _needs_prepare_for_use;
```

Comment:
Oh, this is just a getter and the other method is a pure setter. Can we clean this up rather than having the getting return through a parameter and the setter be parameterless?

Reply:
If any of the apps use this API I volunteer you to clean it up :smile: